policies/recipes/recipe.go

```diff
@@ -0,0 +1,95 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
is this really a new file??

Reply:
yeah.. at first; everything releated to recipe was in recipedb file. i moved anything related to recipe to this file for better readability.

Reply:
i think in the future if you use 'git mv', github will recognize and only show differences, rather than treating this as all new code.